src/tools/mmaps_generator/PathGenerator.cpp

```diff
@@ -244,7 +243,8 @@ int main(int argc, char** argv)
 {
     Trinity::Banner::Show("MMAP generator", [](char const* text) { printf("%s\n", text); }, nullptr);
 
-    int threads = 3, mapnum = -1;
+    unsigned int threads = std::thread::hardware_concurrency();
```

Comment:
There is, if you have threads = 0 even after parameters parsing goes through, nothing happens. It sounds dumb.